test/distributed/org/apache/cassandra/distributed/impl/InternalNodeProbeFactory.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.cassandra.distributed.mock.nodetool;
```

Comment:
I moved the classes out of `mock/nodetool` package to impl, there is no mocking anymore so it is not relevant anymore either